docs/getting-started/getting-started-with-docker.rst

```diff
@@ -22,15 +24,153 @@ There is a good chance that you have heard of containerization. A container is e
+
+Looking for an official image
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+A good place to start when looking for Docker images (or for one to base your own Docker image upon) are official images maintained by institutions. Docker Hub makes this easy by maintaining `a curated list of official images <https://hub.docker.com/search?image_filter=official&q=>`__, and `a curated list of verified images maintained by commerercial entities <https://hub.docker.com/search?q=&image_filter=store>`__. These images include the likes of `Ubuntu <https://hub.docker.com/_/ubuntu>`__, `golang <https://hub.docker.com/_/golang>`__, the `AWS CLI <https://hub.docker.com/r/amazon/aws-cli>`__, and even `Docker <https://hub.docker.com/_/docker>`__. Yes, you can run Docker in Docker!
+
+Quay.io does not maintain a list of official images like Docker Hub, but you can nonetheless find official images on there with a bit of searching, such as `CentOS <https://quay.io/repository/centos/centos?tab=info>`__. Because Quay is owned by Red Hat, it is a good place to find images based on Red Hat Linux distributions.
+
+Looking for an oft-maintained image
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Official images are generally well-maintained. By this we mean that they get frequent updates. Most Docker registries will show you the last time an image was updated, and will show a history going back several updates. This is important not only because it means you can get the latest features of new versions of software, but it also usually indicates that someone is paying attention to the security side of things too. No one wants to use a container only to discover that it still contains `the infamous log4j vulnerability <https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2021-45046>`__, after all.
+
+Security considerations
+~~~~~~~~~~~~~~~~~~~~~~~
+.. note:: Containers operate in an isolated filesystem, and you can *generally* do whatever you like inside of them without affecting the rest of the system. However, it is theoretically possible for malicious software in a container to "escape" into the system at large.
+
+Speaking of log4j, how does one make sure that their images are secure? Even if you think that your research is not a high-value target for hackers, your system might still be useful to someone hoping to mine cryptocurrency, steal credentials, or operate a botnet. Whether you make your own images or use existing ones, there are several things you can do to help keep yourself and your users safe from malicious software. See :ref:`image-container-best-practices` for more information.
```

Comment:
> Speaking of log4j

Seems like a bit of an unnecessary slam on log4j (yeah, there was an issue, but they're not the only library that has had security issues). And it's been a while. I'd just drop the clause and start with `How does one make sure...`